eng/versioning/version_client.txt

```diff
@@ -187,56 +187,56 @@ com.azure:azure-communication-email;1.0.2;1.1.0-beta.1
-com.azure.spring:spring-integration-azure-servicebus;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-integration-azure-storage-queue;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-core;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-actuator-autoconfigure;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-actuator;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-autoconfigure;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-resourcemanager;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-service;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-active-directory;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-active-directory-b2c;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-actuator;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-appconfiguration;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-cosmos;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-data-cosmos;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-eventhubs;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-jdbc-mysql;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-jdbc-postgresql;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-redis;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-keyvault;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-keyvault-certificates;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-keyvault-secrets;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-servicebus-jms;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-servicebus;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-storage;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-storage-blob;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-storage-file-share;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-storage-queue;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-integration-eventhubs;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-integration-servicebus;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-integration-storage-queue;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-stream-eventhubs;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter-stream-servicebus;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-starter;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-stream-binder-eventhubs-core;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-stream-binder-eventhubs;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-stream-binder-servicebus-core;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-stream-binder-servicebus;4.7.0;4.8.0-beta.1
-com.azure.spring:spring-cloud-azure-trace-sleuth;4.7.0;4.8.0-beta.1
+com.azure.spring:spring-cloud-azure-appconfiguration-config-web;4.7.0;4.8.0
```

Comment:
I don't see the spring-data-cosmos here? We suppose to release it with Spring Cloud Azure together.